crates/api/src/lib.rs

```diff
@@ -35,6 +35,15 @@ pub(crate) mod serde_bytes_base64 {
     }
 }
 
+/// A single struct to hold all the API types.
+///
+/// This is for convenience when passing around the API.
+#[derive(Clone)]
+pub struct HostApi {
+    /// The op store.
+    pub op_store: DynOpStore,
+}
```

Comment:
I don't think we have one unified host api in the new kitsune..

I've started [a builder pattern](https://github.com/holochain/kitsune2/pull/19/files#diff-31277162337faccf6affc0143f7bf04374c80b2dae71efef1a00f231b000317bR16) in the peer store PR... but that's for the factories.

The `tx` module instance will be instantiated at the top-level and passed into the individual spaces and most other modules as they are constructed. The spaces themselves will create the op store and peer store instances. Maybe we should just pass the needed instances into the factory functions and let the module implementations figure out how to store them? Or we can provide a `SpaceModuleInstances` struct or something?

Reply:
Gotcha, yeah this was something I started for gossip that might not even be used on this PR. I'll remove it